Automattic/woocommerce-payments
# fix: tokenized cart subscription signup fee price

Fixes #

#### Changes proposed in this Pull Request









I noticed that the pricing for subscriptions with sign-up fee isn't reflected correctly on the tokenized ECE.

The "Order details" dialog just shows the base price, without including any sign-up fees.
This is what a $3/month subscription w/ a $1 sign-up fee looks like - you'll notice that the "total" price is accurate, but the product price isn't accounting for the sign-up fee:
<img width="596" alt="Screenshot 2024-12-18 at 11 28 52 AM" src="https://github.com/user-attachments/assets/61d0576e-ac08-49b0-8594-3d1c4192ceca" />

After:
<img width="596" alt="Screenshot 2024-12-18 at 11 33 07 AM" src="https://github.com/user-attachments/assets/1c79c268-4862-4dc3-984b-3cd42fccf01e" />


Fixing.

#### Testing instructions





- Ensure you have the "Enable Cart-Token implementation for ECE" flag enabled in your WCPay Dev settings
- Ensure you have a subscription product w/ a sign-up fee (or create one if needed)
- Go to the product page for the product
- Click on the GooglePay button
- Click on the ℹ️ icon next to the price
- The product price should be inclusive of the sign-up fee

-------------------


**Post merge**
